src/main/scala/mesosphere/marathon/api/BeanValidation.scala

```diff
@@ -76,3 +76,5 @@ trait BeanValidation {
   }
 }
 
+object BeanValidation extends BeanValidation
```

Comment:
No but in my opinion it should.

I do not like using traits like this. Do you?